src/initializers/diamorphosis.ts

```diff
@@ -275,6 +276,11 @@ function addGrowthbookConfig(config) {
   config.growthbook = {
     apiHost: 'https://cdn.growthbook.io',
     clientKey: '',
+    setAttributesCallback: setAttributesCallback,
```

Comment:
🙃 I would move this to https://github.com/Workable/orka/blob/024156548971dee9d997dd403f2bd09b2e6b560e/src/default-options.ts#L41 because we don't have functions in config usually.
No strong opinion though.